src/app/containers/MostRead/index.jsx

```diff
@@ -30,6 +60,12 @@ const MostReadContainer = ({
     return null;
   }
 
+  // Do not render most read on idx page when toggle on idx-pages is disabled
+  const idxPageHasMostRead = pathOr(false, ['hasMostRead'], idxPage);
+  if (isOnIdxPage && !idxPageHasMostRead) {
+    return null;
+  }
+
```

Comment:
Could we have `const mostReadToggleEnabled =
    (enabled && hasMostRead) || (enabled && idxPageHasMostRead);` as opposed to introducing a `isOnIdxPage` prop?

Reply:
We only want to check the `idxPageHasMostRead` toggle if we're on an IDX page. 

`const mostReadToggleEnabled = (enabled && hasMostRead) || (enabled && idxPageHasMostRead);` this logic might cause mostread do be displayed on frontpages if `hasMostRead` is turned off but `idxPageHasMostRead` is on.

Reply:
Could we check config on the IDX page, rather than in this container? And then only render the most read container if we want to show it on the page we are on

Reply:
I think the approach @tochwill mentioned is the same way the stories team did their approach with using Most Read in their STY Pages. I think it might be worth seeing if we can take any more from their approach for consistency.

Reply:
@tochwill yes we can but that will be different from how we check the config for most other page-types